easy-ui-react/src/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,94 @@
+import React, {
+  MutableRefObject,
+  ReactNode,
+  useMemo,
+  useRef,
+  useState,
+} from "react";
+import { useEdgeInterceptors } from "../DataGrid/useEdgeInterceptors";
+import { getComponentToken } from "../utilities/css";
+import { Edge } from "./Edge";
+import { TabsItem } from "./TabsItem";
+import { TabsContext } from "./context";
+import { useScrollbar } from "./useScrollbar";
+
+import styles from "./Tabs.module.scss";
+
+export type TabsProps = {
+  containerComponent: "div" | "nav";
+  containerProps: object;
+  listComponent: "div" | "ul";
+  listProps: object;
+  listRef?: MutableRefObject<null>;
+  children: ReactNode;
+};
+
+/**
+ * Do not use this component directly.
+ *
+ * `Tabs` is the underlying implementation to the presentation of tabs in
+ * `TabNav` and `TabPanels`. The specifics of each implementation is handled
+ * by the respective component.
+ */
```

Comment:
**question**: not super important, but do you know if theres a doc tag or something we might be able to use here to indicate that it shouldn't be used directly?

Reply:
good call. i updated to use `@private` and `@ignore` jsdoc tags to signify it's private (slight abuse of that tag but it's fine) and `ignore` for not being a part of public documentation